src/commands/force/user/display.ts

```diff
@@ -34,7 +34,11 @@ export class UserDisplayCommand extends SfdxCommand {
 
   public async run(): Promise<Result> {
     this.logger = await Logger.child(this.constructor.name);
-
+    if (sfdc.matchesAccessToken(this.flags.targetusername)) {
+      throw new SfdxError(messages.getMessage('accessTokenError'), 'accessTokenError', [
```

Comment:
should be equivalent, but just for your knowledge you can also do `SfdxError.create('plugin-user', 'create', 'accessTokenError', [tokens])`